### DISCUSSION TITLE
Trying to make a hattire

### DISCUSSION THREAD

**Discussion by YousefMohamed101** [opentrack/opentrack#1507] [Q&A]
hello so am trying to make a custom hattire code for my knockoff mpu9250 since i cant get my hands on the original one and not library i searched wanted to work with my mpu any help is appreciated

**Comment by GO63-samara** [comment]
Try this:
https://github.com/opentrack/opentrack/issues/1271

Or this ( SensorVariant 8 ):
https://sites.google.com/site/diyheadtracking/main/5dof-tracker
